packages/graphql-server-integration-testsuite/src/index.ts

```diff
@@ -457,6 +469,29 @@ export default (createApp: CreateAppFunc, destroyApp?: DestroyAppFunc) => {
           });
       });
 
+       it('can handle batch requests in parallel', function() {
```

Comment:
yeah i thought about creating a promise and resolve it on the other query.
but it felt just too bad..
this is probably what i would've done if it was not going through the whole query engine and such.